metrics/grafana/tikv_details.json

```diff
@@ -6000,7 +6000,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(rate(tikv_thread_cpu_seconds_total{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", name=~\"raftstore_.*\"}[1m])) by (instance)",
```

Comment:
There is also a file named "tikv_summary.json" needs to be updated.

Reply:
They are already correct.